app/views/search/index.html.erb

```diff
@@ -18,41 +17,58 @@
 </div>
 
 <%= form_tag(search_path, method: :get) do %>
-    <div class="input-group mb-3">
-        <%= text_field_tag :search, params[:search], class: "form-control" %>
+    <div class="input-group mb-3 mt-3">
+        <%= text_field_tag :search, params[:search], class: "form-control", placeholder: "Suche" %>
         <button type="submit" class="btn btn-outline-secondary">
             <i class="bi bi-search"></i>
         </button>
         <button type="button" class="btn btn-outline-secondary" data-bs-toggle="modal" data-bs-target="#qrscanner">
             <i class="bi bi-qr-code-scan"></i>
         </button>
     </div>
-    <div class="btn-group mb-3">
-        <button class="btn btn-secondary dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false">
-            Filter
-        </button>
-        <div class="dropdown-menu">
-            <div class="input-group">
-                <%= select_tag "filter_property", raw("<option>Title</option><option>Author</option><option>Release</option>"), class: "form-select" %> <%# https://apidock.com/rails/ActionView/Helpers/FormTagHelper/select_tag %>
-                <%= text_field_tag :filter_value, params[:filter_value], class: "form-control" %>
+    
+    <button class="btn btn-secondary" type="button" data-bs-toggle="modal" aria-expanded="false" data-bs-target="#filter-modal">
+        FILTER <i class="bi bi-sliders2-vertical"></i>
+    </button>
+    
+    <div class="modal-fullscreen modal fade" id="filter-modal" tabindex="-1" aria-labelledby="filter-modal-title" aria-hidden="true">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h1 class="modal-title fs-5" id="filter-modal-title">Modal title</h1>
+                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                </div>
+                <div class="modal-body">
+                    <div class="input-group">
+                        <%= select_tag "filter_property", raw("<option>Title</option><option>Author</option><option>Release</option>"), class: "form-select" %> <%# https://apidock.com/rails/ActionView/Helpers/FormTagHelper/select_tag %>
```

Comment:
Thank you for the link to the documentation 🥳 It would be very interesting to know, why do you use `raw` instead of `options_from_collection_for_select`. Maybe it's a good idea to get the collections of options from the controller. What do you think?

Reply:
As it is currently unclear to me, how the filters will work (and we were just testing anyway) we just used `select_tag raw`. Your idea sounds good!